src/pages/mypage/hooks/useDetectionPrefetch.client.ts

```
@@ -0,0 +1,323 @@
+import { buildHotspotsPipeline } from '@pages/generate/hooks/furnitureHotspotPipeline';
+import { loadCorsImage } from '@pages/generate/hooks/useFurnitureHotspots';
+import { useONNXModel } from '@pages/generate/hooks/useOnnxModel';
+import { useDetectionCacheStore } from '@pages/generate/stores/useDetectionCacheStore';
+import {
+  filterAllowedDetectedObjects,
+  mapHotspotsToDetectedObjects,
+} from '@pages/generate/utils/detectedObjectMapper';
+
+import type { FurnitureCategoryCode } from '@pages/generate/constants/furnitureCategoryMapping';
+import type { FurnitureHotspot } from '@pages/generate/hooks/useFurnitureHotspots';
+import type { ProcessedDetections } from '@pages/generate/types/detection';
```

ESLint

[error] There should be no empty line within import group

(import/order)